src/Internal/Payment/State/AbstractPaymentState.php

```diff
@@ -91,17 +91,43 @@ protected function create_state( string $state_class ) {
 	 */
 
 	/**
-	 * Initialtes the payment process.
+	 * Initiates the payment process.
 	 *
 	 * @param PaymentRequest $request    The incoming payment processing request.
-	 * @return CompletedState            The next state.
+	 * @return AbstractPaymentState      The next state.
 	 * @throws StateTransitionException  In case the completed state could not be initialized.
 	 * @throws ContainerException        When the dependency container cannot instantiate the state.
 	 * @throws Order_Not_Found_Exception Order could not be found.
 	 * @throws PaymentRequestException   When data is not available or invalid.
 	 * @psalm-suppress InvalidReturnType If this method does not throw, it will return a new state.
 	 */
-	public function process( PaymentRequest $request ) {
+	public function start_processing( PaymentRequest $request ) {
+		$this->throw_unavailable_method_exception( __METHOD__ );
+	}
+
+	/**
+	 * Process verified state.
+	 *
+	 * @psalm-suppress InvalidReturnType
+	 *
+	 * @return AbstractPaymentState
+	 * @throws \WCPay\Exceptions\Order_Not_Found_Exception
+	 * @throws \WCPay\Internal\Payment\Exception\StateTransitionException
+	 */
+	public function process() {
```

Comment:
This method is not used anywhere, let's only leave `start_processing` here.

Reply:
Removed (https://github.com/Automattic/woocommerce-payments/pull/7471/commits/48da0fc92461c2e53787cf45c2e79dcf0fa5bf8d)